site/content/docs/cli/npm-packages.md

```diff
@@ -26,4 +29,58 @@ The latest runtime is `2022.10` at the time of writing. This runtime contains am
 - [Browse the latest runtime specs](https://www.checklyhq.com/docs/runtimes/specs/)
 - [Learn more about runtimes](https://www.checklyhq.com/docs/runtimes/)
 
+## Local Dependencies
+
+Browser Checks are also able to import other JavaScript and TypeScript files as dependencies.
+This can be particularly useful for defining helper functions to be reused across multiple checks.
+The Checkly CLI will automatically detect these dependencies and make sure that they're bundled as part of the check.
+No additonal configuration is needed, and there's no need to create [Snippets](/docs/snippets/) in your account.
+
+Here is an example of how this works in practice. The directory tree looks like the following:
+```
+__checks__
+├── login.check.ts
+├── login.spec.ts
+└── login-helper.ts
+```
+
+In `login-helper.ts` we define a reusable function `gitHubLogin`. This function can be used by multiple Browser checks.
+```
+export async function gitHubLogin (page, username, password) {
+  await page.goto('https://github.com/login')
+  await page.getByLabel('Username or email address').type(username)
+  await page.getByLabel('Password').type(password)
+  await page.getByRole('button', { name: 'Sign in' })
+}
+```
+
+In `login.spec.ts` we define the actual Playwright test. This file can import the `gitHubLogin` function from `login-helper.ts`. It also reads the username and password from [remote environment variables](https://www.checklyhq.com/docs/cli/using-environment-variables/#remote-environment-variables).
+```
+// @ts-ignore
+import { test } from '@playwright/test'
+import { gitHubLogin } from './login-helper'
+
+test('Github login', async ({ page }) => {
+  await gitHubLogin(page, process.env.GITHUB_USER, process.env.GITHUB_PWD)
+
+  // your normal check code
+  await page.click('.header-search-input')
+})
+```
+
+Finally, in `login.check.ts` we define the [Browser Check construct](/docs/cli/constructs/#browsercheck). Note that it's only necessary to configure the main Playwright file `login.spec.ts`. The dependency on `login-helper.ts` is automatically detected by the CLI.
+```
+import { BrowserCheck } from '@checkly/cli/constructs'
+
+new BrowserCheck('login-check', {
+  name: 'Login Check',
+    code: { entrypoint: './login.spec.ts' }
+  })
+})
+```
+
+After running [`npx checkly deploy`](/docs/cli/command-line-reference/#npx-checkly-deploy) to deploy the project, we can see in the Web UI that the helper file `login-helper.ts` was also uploaded for the check.
+
+![login check with helper file in dependencies](/docs/images/cli/github_login_helper_dependency.png)
```

Comment:
@clample take this as an optional suggestion, but I think the screenshot could use either a zoom in to the dependency bit, or a callout of some sort highlighting it

Reply:
Agreed as optional, but a leading focus would be 💯 . Example (https://checklyhq.slack.com/archives/C03H435FU5R/p1677098383180029?thread_ts=1677095180.409809&cid=C03H435FU5R)